event-grid/subscribe-to-blob-storage/subscribe-to-blob-storage.ps1

```diff
@@ -4,19 +4,22 @@ $storageName = "contosostorage"
 # Provide an endpoint for handling the events.
 $myEndpoint = "<endpoint URL>"
```

Comment:
Endpoints have a specific required format ('https://'). Unclear if this should be documented in the comments or not.